README.md

```diff
@@ -12,8 +12,55 @@ These instructions will enable you to get the [Full Scan](https://github.com/zap
 ### Pre-requisites
 Docker needs to be installed on the machine the agent will be running on.
 
-### The YAML file explained
-This first stage ensures that the agent has access to the pre-defined $(Build.ArtifactStagingDirectory) location, this may or may not be needed - this is used here as we are running on Linux.
+### Incorporate into an Azure DevOps Pipeline - UKHO
+This can be incorpated into an Azure Devops Pipeline by copying the [azure-pipelines.yml](https://github.com/UKHO/owasp-zap-ui-scan/blob/master/azure-pipelines.yml) and using this within a pipeline created against your repository for running the Zap scan. 
+
+All that needs to be done is to add a pipeline variable called **ApplicationUrl**, which will be the base URL of the application under test.
+
+    resources:
+	    repositories:
+		    - repository: owaspzapui
+		      type: github
+		      endpoint: UKHO
+		      name: UKHO/owasp-zap-ui-scan
+		      ref: refs/heads/master
+	jobs:
+	- template: owasp-zap-ui-scan-template.yml@owaspzapui
+	  parameters:
+	  url: $(ApplicationUrl)
+	  
+This yaml will use the contents of the master branch for this repository, using [owasp-zap-ui-scan-template.yml](https://github.com/UKHO/owasp-zap-ui-scan/blob/master/owasp-zap-ui-scan-template.yml), [ZapTransform.ps1](https://github.com/UKHO/owasp-zap-ui-scan/blob/master/src/ZapTransform.ps1) and [ZapTransformTemplate.xslt](https://github.com/UKHO/owasp-zap-ui-scan/blob/master/src/ZapTransformTemplate.xslt) on the master branch of this repository. 
```

Comment:
You mention this twice "on the master branch of this repository. "

Reply:
Removed 👍